packages/appstore/src/components/CFDs/cfd-demo-accounts.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { localize } from '@deriv/translations';
+import { CFD_PLATFORMS } from '@deriv/shared';
+import AccountManager from '../account-manager';
+import { TCFDAccountsProps, TPlatform, TDetailsOfEachMT5Loginid } from 'Types';
+
+const CFDDemoAccounts = ({ isDerivedVisible, isFinancialVisible, current_list }: TCFDAccountsProps) => {
+    const available_demo_accounts = [
+        {
+            name: 'Derived',
+            description: localize('Trade CFDs on MT5 with Derived indices that simulate real-world market movements.'),
+            is_visible: isDerivedVisible(CFD_PLATFORMS.MT5),
+            disabled: false,
+            platform: CFD_PLATFORMS.MT5,
+            type: 'synthetic',
+        },
+        {
+            name: 'Financial',
+            description: localize('Trade CFDs on MT5 with forex, stocks & indices, commodities, and cryptocurrencies.'),
+            is_visible: isFinancialVisible(CFD_PLATFORMS.MT5),
+            disabled: false,
+            platform: CFD_PLATFORMS.MT5,
+            type: 'financial',
+        },
+        {
+            name: 'Deriv X',
+            is_visible: isDerivedVisible(CFD_PLATFORMS.DXTRADE),
+            description: localize(
+                'Trade CFDs on Deriv X with Derived indices, forex, stocks & indices, commodities and cryptocurrencies.'
+            ),
+            has_account: false,
+            disabled: false,
+            platform: CFD_PLATFORMS.DXTRADE,
+            type: 'synthetic',
+            // ToDo: deriv x should have type of all in new API
```

Comment:
For Dxtrade we will have `type='all' ` instead of `synthetic` and `financial`, we will change this accordingly